Source/DataSources/NetworkLinkUpdateMgr.js

```diff
@@ -0,0 +1,123 @@
+import defined from "../Core/defined.js";
+
+/**
+ * NetworkLinkUpdateMgr keeps track kmls that are loaded from network links so that
+ * they can be updated later.
+ *
+ * When an 'Update' link is loaded, the manager will use the link's targetHref to
+ * find the stored kml and updated it.
+ */
+export default function NetworkLinkUpdateMgr() {
+  this._links = {};
+  this._sourceToUrl = {};
```

Comment:
Switched the old `_kmlSources` for `_links` (same functionality)
and added _sourceToUrl for `destroy`